sage-ios/SAGE/SAGE/AddSchoolRadiusViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  AddSchoolRadiusViewController.swift
+//  SAGE
+//
+//  Created by Erica Yin on 2/21/16.
+//  Copyright © 2016 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+
+class AddSchoolRadiusViewController: UIViewController {
+    
+    var schoolRadiusView: AddSchoolRadiusView?
```

Comment:
doesnt need to be optional just initialize it here and in loadView just say self.view = self.schoolRadiusView